WordPress/Classes/Networking/MediaHost+Blog.swift

```diff
@@ -8,6 +8,13 @@ extension MediaHost {
         case baseInitializerError(error: Error, blog: Blog)
     }
 
+    init(with blog: Blog) {
+        self.init(with: blog) { error in
+            // We'll log the error, so we know it's there, but we won't halt execution.
+            WordPressAppDelegate.crashLogging?.logError(error)
+        }
+    }
+
     init(with blog: Blog, failure: (BlogError) -> ()) {
```

Comment:
Have you considered using argument default for this over a dedicated init?

E.g.

```swift
init(
    with blog: Blog,
    failure: (BlogError) -> () = { error in
        // ...
    }
) {
```

Granted this approach assumes that the logging mechanism will remain a singleton shared instance, which might not be desirable in the future.